docs/game_sdk/Discord.md

```diff
@@ -126,14 +126,14 @@ You should begin your integration by setting up this callback to help you debug.
 | 26   | NotInstalled                    | Discord is not installed                                                                        |
 | 27   | NotRunning                      | Discord is not running                                                                          |
 | 28   | InsufficientBuffer              | insufficient buffer space when trying to write                                                  |
-| 29   | PurchaseCancelled               | user cancelled the purchase flow                                                                |
+| 29   | PurchaseCancelled               | the user, canceled the purchase flow                                                            |
```

Comment:
user cancelled the purchase flow is correct. Comma is unnecessary.

Reply:
@Lachee Could you start a review using suggestions (Ctrl-G on the lines)? That would make applying the changes a lot easier! 👍  And thanks for reviewing this

Reply:
should be "user canceled the purchase flow"